Kickstarter-iOS/Features/MessageBanner/Views/MessageBannerView.swift

```diff
@@ -3,24 +3,31 @@ import SwiftUI
 
 struct MessageBannerView: View {
   @Binding var viewModel: MessageBannerViewViewModel?
+  @SwiftUI.Environment(\.accessibilityVoiceOverEnabled) private var voiceOverEnabled
 
   var body: some View {
     if let vm = viewModel {
-      ZStack {
-        RoundedRectangle(cornerRadius: 4)
-          .foregroundColor(vm.bannerBackgroundColor)
-        Label(vm.bannerMessage, image: vm.iconImageName)
+      HStack(alignment: .center, spacing: 12) {
+        Image(vm.iconImageName, bundle: Bundle.framework)
+        Text(vm.bannerMessage)
           .font(Font(UIFont.ksr_subhead()))
-          .foregroundColor(vm.messageTextColor)
-          .lineLimit(3)
           .multilineTextAlignment(vm.messageTextAlignment)
-          .padding(EdgeInsets(top: 5, leading: 10, bottom: 5, trailing: 10))
+          .frame(
+            maxWidth: .infinity,
+            minHeight: 40,
+            alignment: self.alignmentFromTextAlignment(vm.messageTextAlignment)
+          )
+      }
+      .foregroundColor(vm.messageTextColor)
+      .padding(EdgeInsets(top: 16, leading: 9, bottom: 16, trailing: 9))
+      .background {
+        RoundedRectangle(cornerRadius: 6)
```

Comment:
This corner radius (and possibly the padding) matches the figma for PPO but is different from the UIKit banners. I'm going to confirm with Alison if we want to match the figma or if we want to be consistent with the app before I submit the pr.

Reply:
What do you think about having some kind of MessageBannerViewLayout enum to avoid hardcoding the padding, cornerRadius, minHeight, and spacing values?

Reply:
Done! I wanted to do a shared enum between the swiftUI and UIKit versions, but the UIKit version is primarily a storyboard, so I gave up on that pretty quickly... I've also changed some of the values (and re-recorded screenshots) after talking to Alison and determining that PPO styling should match current standards, not our Figma mocks.